### PR TITLE
email feature added

### PR DESCRIPTION
**Pull request by dlowrey**
Implemented email functionality in `mailer.js`, 'src/appointments/manager.js`, and 'src/appointments/db.js`.

Invoice Email sent to patient upon completion of appointment. 
Our email address: `fakehealthcareforyou@gmail.com`
password: ask me

### CODE REVIEW

File: src/mailer.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const sendInvoice = (data) => {
+
+  let body = "Hello, and thank you for your recent visit to " +
+             "team null's hospital. We hope you are satisfied with your " +
+             "placebo treatment, we are sure you will be feeling better in " +
+             "null time. Now for the stuff we really care about: " +
+             "\n\nYour invoice total comes out to: $" + data.amount + ".\n\n" +
+             "You can pay this using your online invoice number: " +
+             data.id + " at [insert endpoint here]." +
```
Comment (NilsG-S) [inline src/mailer.js:20]: Is `insert endpoint here` supposed to be a link to the invoice page?

**Review by NilsG-S — CHANGES_REQUESTED** [review]
Could you add some documentation to the server setup reminding me to set the password?

File: src/mailer.js (hunk 1 above)
Comment (dlowrey) [reply src/mailer.js:20]: @NilsG-S Yes! I wasn't sure what it was.

Comment (NilsG-S) [reply src/mailer.js:20]: @dlowrey Alright, well I haven't setup the hosting for the webapp, so I can't give you a link yet. But we can make the change later

**Review by NilsG-S — APPROVED** [review]

**Pull request merged**